sysl2/sysl/sysl.go

```diff
@@ -15,12 +15,17 @@ import (
 	"gopkg.in/alecthomas/kingpin.v2"
 )
 
+// Version contains the sysl binary version
+//nolint:gochecknoglobals
+var Version = "unspecified"
```

Comment:
Version should be a const.

Reply:
Although it sounds like a constant, cannot use constants as value should be reassigned during compile time